src/libs/actions/PaymentMethods.js

```diff
@@ -164,6 +160,20 @@ function saveWalletTransferAmount(transferAmount) {
     Onyx.merge(ONYXKEYS.WALLET_TRANSFER, {transferAmount});
 }
 
+/**
+ * @param {String} selectedAccountID
+ */
+function saveWalletTransferAccount(selectedAccountID) {
+    Onyx.merge(ONYXKEYS.WALLET_TRANSFER, {selectedAccountID});
+}
+
+/**
+ * @param {String} filterPaymentMethodType
+ */
```

Comment:
This doc could use a description because it's not clear what the `filterPaymentMethodType` is used for

>Toggles the user's selected type of payment method (bank account or debit card) on the wallet transfer balance screen.

Reply:
Sure